.github/workflows/sycl-windows-build.yml

```diff
@@ -164,6 +165,19 @@ jobs:
       if: always() && !cancelled() && contains(inputs.changes, 'libdevice')
       run: |
         cmake --build build --target check-libdevice
+    - name: Generate/diff new ABI symbols
+      if: always() && !cancelled() && contains(inputs.changes, 'sycl')
+      shell: bash
+      run: |
+        LLVM_BIN_PATH="build/bin" python.exe src/sycl/tools/abi_check.py --mode dump_symbols --output build/new_sycl_symbols_windows.dump build/bin/sycl?.dll
+        diff -Naur src/sycl/test/abi/sycl_symbols_windows.dump build/new_sycl_symbols_windows.dump || true
+    - name: Upload new ABI symbols
+      if: always() && !cancelled() && contains(inputs.changes, 'sycl')
```

Comment:
lgtm but can you test this in postcommit and make sure the resullts are what you expect? we use `icx` there and that might do something different/dumb

Reply:
https://github.com/intel/llvm/actions/runs/13592706944

Reply:
ill trust you to check that it passes :) thx

Reply:
Build passed, merging.